src/redux-js/actions/LoginAction.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import axios from 'axios';
+import Login, {userLoginFail} from './LoginActionCreator';
+import {notify} from 'react-notify-toast';
+
+export const userLoginRequest = (user) => {
+	return async dispatch => {
+		return await axios.post('https://ah-backend-staging.herokuapp.com/api/users/login/', {user}
```

Comment:
Please make use of the constants file and define all `URLs` there.

Reply:
Thanks for pointing out. I have handled this.